lib/View/Matcher/SyliusResource/Instance.php

```diff
@@ -0,0 +1,27 @@
+<?php
+
+declare(strict_types=1);
+
+namespace Netgen\Layouts\Sylius\View\Matcher\SyliusResource;
+
+use Netgen\Layouts\Sylius\View\View\SyliusResourceViewInterface;
+use Netgen\Layouts\View\Matcher\MatcherInterface;
+use Netgen\Layouts\View\ViewInterface;
+
+class Instance implements MatcherInterface
+{
+    public function match(ViewInterface $view, array $config): bool
+    {
+        if (!$view instanceof SyliusResourceViewInterface) {
+            return false;
+        }
+
+        foreach ($config as $fqcn) {
```

Comment:
Can we use `$className` instead of `$fqcn` here. No need for shortcuts in variable names.

Reply:
@emodric Fixed in https://github.com/netgen-layouts/layouts-sylius/pull/14/commits/78c11112af30a4e97f7baffde7382c8f5c593ecd